### ISSUE TITLE
[Accessibility] Narrator is not announcing values of "Multi-select uncontrolled" ComboBox

### ISSUE THREAD

**Issue by SyedRehmanRazvi** [OfficeDev/office-ui-fabric-react#10910]
User Experience:
Users who rely on screen readers are getting affected as narrator is not announcing values of "Multi-select uncontrolled" dropdown.

Repro Steps:
Step 1: Start Narrator
Step 2: Open Fabric UI URL in Edge browser.
Step 3: Navigate to "Multi-select uncontrolled" dropdown available in "Dropdown" page and activate it
Step 4: Select 2 or mode check boxes.
Step 5: Verify whether narrator is announcing values of "Multi-select uncontrolled" dropdown.

Actual:
Narrator is not announcing values of "Multi-select uncontrolled" dropdown
It is announcing as “Multi-select combo box selected"

Expected:
Narrator should announce values of "Multi-select uncontrolled" dropdown
It should announce as “Multi-select combo box selected Option C, Option D”

![image](https://user-images.githubusercontent.com/53210862/67110176-bc4ab080-f1ef-11e9-88d1-71ab373990e8.png)

**Comment by marygans** [comment]
Hi @SyedRehmanRazvi  thank you for submitting this issue!

I was able to repro this issue. The selected options in the input box disappears on focus which is also not expected behavior. I will further investigate this issue. 

Thanks again for submitting.
